F4 profile

Jun-How opened the pull request:
Implemented the UI design of the User Profile feature.

Dangabit commented:
A few things to improve on the UI:
- Missed out on "forget password" UI
For web version:
- Sign out button floating
- A few text were in the middle of nowhere
- Super long alertdialog

Meanwhile, I have improved some functions and fixed some bugs I found along the way.

Jun-How commented:
- Added simple UI for "forget password"
- Fixed the misalignments of buttons and texts showing on web screen
- Shortened alertdialog on web screen
- Currently using a temporary solution for responsive design based on simple size configuration, will look more in depth for better solutions after Milestone 2.
